src/infinite-list/index.tsx

```diff
@@ -44,7 +44,7 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
     scrollViewProps,
     positionIndex = 0
   } = props;
-  
```

Comment:
Please remove the changes here. There is no need to include this file if there are no actual changes here